src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -70,8 +89,7 @@ class SettingsMigration : Activity {
             dotcomAccessToken,
             server,
             requestExecutorFactory,
-            EmptyProgressIndicator(ModalityState.NON_MODAL),
-            customRequestHeaders)
+            EmptyProgressIndicator(ModalityState.NON_MODAL))
```

Comment:
What that change does in practice?

Reply:
I removed `id` parameter from `addAsDefaultAccountIfUnique`. I don't know why here `customRequestHeaders` were passed as id  but it should be always fetched from `UserDetails`.